doc/advanced-configuration/Listener-modules.md

```diff
@@ -34,7 +34,8 @@ Manages all HTTP-based services, such as BOSH (HTTP long-polling) and WebSocket.
 
 * `ip` (IP tuple, optional, default: `{0,0,0,0}`) - IP address to bind to.
 * `num_acceptors` (positive integer, optional, default: 100) - Number of acceptors.
-* `max_connections` (positive integer, optional, default: 1024) - Maximum total number of HTTP(S) connections.
```

Comment:
It would be great to still support this option in the code (but still remove it from documentation) to preserve backwards compatibility for documented options throughout minor changes.